CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 - Fix definition conflicts with previous value
   - https://github.com/Shopify/flash-list/pull/795
+- Fix Android unit test
+  - https://github.com/Shopify/flash-list/pull/815
```

Comment:
I don't think we need this in the changelog since this is not a user-facing change.